### ISSUE TITLE
M1 kmutil load error : "Extension with identifiers xxx not approved to load" when sip is already disable

### ISSUE THREAD

**Issue by baymaxChen1** [saagarjha/TSOEnabler#6]
sudo kmutil load -v -p /Library/Extensions/BaymaxKernelExtension.kext.copy.kext
<img width="686" alt="image" src="https://user-images.githubusercontent.com/85376176/120874448-5041d980-c5d9-11eb-9041-09d5483ac3e0.png">

In System Preference, it suggests that `enable kernel extensions from the Security Policy `, but my M1 device is already disable sip
<img width="411" alt="image" src="https://user-images.githubusercontent.com/85376176/120874486-72d3f280-c5d9-11eb-9265-6de6bc5cb823.png">

<img width="217" alt="image" src="https://user-images.githubusercontent.com/85376176/120874554-ac0c6280-c5d9-11eb-8aa7-1cb0f88686bd.png">

**Comment by saagarjha** [comment]
I would suggest turning on SIP and upping your boot policy to maximum, then dropping both back down again. I've seen cases where they don't "stick" properly.

**Comment by baymaxChen1** [comment]
> I would suggest turning on SIP and upping your boot policy to maximum, then dropping both back down again. I've seen cases where they don't "stick" properly.

I have tried `csrutil enable` -> `csrutil enable --without debug --without kext` | `csrutil disable` before, but it doesn't work.
Do you succeed running kernel extension in M1 without  KEXT signing certificate?

**Comment by saagarjha** [comment]
Yes, I have TSOEnabler loaded with a normal development certificate.
